### PR TITLE
[TT-985] [TT-986] Release 3.0.2

### PR DESCRIPTION
**Pull request by marksou**
[TT-986] Refactor do not track plugin documentation and [TT-985] Add new Redis configuration options documentation to components

[TT-986]: https://tyktech.atlassian.net/browse/TT-986
[TT-985]: https://tyktech.atlassian.net/browse/TT-985

### CODE REVIEW

File: tyk-docs/content/tyk-configuration-reference/redis-cluster.md [hunk 1]
```diff
@@ -151,6 +151,19 @@ To enable a Redis Sentinel setup from v2.9.3 onwards, you need to set the Master
 * In the Tyk Pump config file - `storage.master_name`
 * In a MDCB installation config file - `storage.master_name`
 
+### Support for Redis Sentinel AUTH
+
+To support the use of Redis Sentinel AUTH (introduced in Redis 5.0.1) we have added the following global config settings in Tyk v3.0.2:
+
+* In the Tyk Gateway config file - `sentinel_password`
+* In the Tyk Dashboard config file - `redis_username` and `redis_sentinel_password`
```
Comment (furkansenharputlu) [inline tyk-docs/content/tyk-configuration-reference/redis-cluster.md]: `username` parameters are not related to redis sentinel. 

For example: `redis_username` for dashboard was missing in dashboard redis configuration. It is added but it is not related to sentinel.

Comment (marksou) [reply tyk-docs/content/tyk-configuration-reference/redis-cluster.md]: I've made some amendments.

**Review by furkansenharputlu — APPROVED** [review]

File: tyk-docs/content/tyk-configuration-reference/redis-cluster.md [hunk 2]
```diff
@@ -151,6 +151,19 @@ To enable a Redis Sentinel setup from v2.9.3 onwards, you need to set the Master
...
+
+To support the use of Redis Sentinel AUTH (introduced in Redis 5.0.1) we have added the following global config settings in Tyk v3.0.2:
+
+* In the Tyk Gateway config file - `sentinel_password`
+* In the Tyk Dashboard config file - `redis_sentinel_password`
+* In the Tyk Pump config file - `sentinel_password`
+* In the Tyk Identity Broker config file - `SentinelPassword`
+* In the Tyk Synk config file - `sentinel_password`
+
+These settings allow you to support Sentinel password-only authentication in Redis version 5.01 and above.
```
Comment (furkansenharputlu) [inline tyk-docs/content/tyk-configuration-reference/redis-cluster.md]: ... Redis version 5.0.1 ...

Comment (marksou) [reply tyk-docs/content/tyk-configuration-reference/redis-cluster.md]: Fixed

**Pull request merged**